sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -78,7 +90,15 @@ function createPropertiesFromSpan(span: ReadableSpan): [Properties, Measurements
   const measurements: Measurements = {};
 
   for (const key of Object.keys(span.attributes)) {
-    if (!(key.startsWith("http.") || key.startsWith("rpc.") || key.startsWith("db."))) {
+    if (
+      !(
+        key.startsWith("http.") ||
+        key.startsWith("rpc.") ||
+        key.startsWith("db.") ||
+        key.startsWith("peer.") ||
+        key.startsWith("net.")
+      )
+    ) {
       properties[key] = span.attributes[key] as string;
```

Comment:
Truncation rules for [custom properties](https://github.com/microsoft/common-schema/blob/main/Mappings/AzureMonitor-AI.md#mapping-rule) need to be applied. `_MS.links` needs to be a valid JSON after truncation.

Reply:
So you need to parse the JSON and remove links, stringify it again and keep trying until size is allowed, will address in different PR

Reply:
Since links are a fixed length, we can do some math and allow only up to a certain amount of links :D